### ISSUE TITLE
getReadableVersion() and getVersion() returning "null.0" and null respectively

### ISSUE THREAD

**Issue by shikhar91939** [react-native-community/react-native-device-info#676]
The above mentioned two methods are working correctly for iOS but returning null values for Android.
```
import DeviceInfo from 'react-native-device-info';

...

getDeviceInfo = () => {
    return {
      appBuildNumber: DeviceInfo.getBuildNumber(),
      appVersion: DeviceInfo.getReadableVersion(),
      getVersion: DeviceInfo.getVersion(),
      getApplicationName: DeviceInfo.getApplicationName(),
      hardware: DeviceInfo.getDeviceId(),
      deviceOSVersion: DeviceInfo.getSystemVersion(),
      manufacturer: DeviceInfo.getManufacturer(),
      brand: DeviceInfo.getBrand(),
    };
  }
```
Is something wrong with the way that I am using it ?
Can I change something in this repo to fix the issue ?

**Comment by mikehardy** [comment]
Check the example app - those are working fine, so I suspect a project-local issue (link failure usually). If the example app returns null then maybe we have a device-specific issue and we can reopen and triage

**Comment by shikhar91939** [comment]
Hey @mikehardy, thank you for your response. I am pretty new to React Native. Can you tell me what you mean by link failure. I tried to google it but did not get anything useful.

**Comment by mikehardy** [comment]
@shikhar91939  welcome to react-native then :-), and your first issues getting native modules working - I believe it is a rite of passage, so you now you are a true react-native developer haha. I mean that (assuming npm for these examples, yarn would work too but slightly different commands) -

* `npm install react-native-device-info` is first to install the module, then you:
* `npm link react-native-device-info` to integrate the module into your Java/iOS projects (at least "auto-linking" coming in react-native 0.60 is out and implemented, for 0.59 and lower, you must link)
* then the import in the actual javascript implementation, which seems correct

If you look at [refresh-example.sh](https://github.com/react-native-community/react-native-device-info/blob/master/refresh-example.sh) you'll see an example of how to automatically construct an example app, all linked in. Then the example itself (which is just the committed app built from refresh-example you can see it running, and it does work

**Comment by shikhar91939** [comment]
Thanks a lot for the help @mikehardy !
